src/lib/rawClass.ts

```diff
@@ -61,6 +61,10 @@ export type RawClass = {
   u2: number;
   /** Outside class units */
   u3: number;
+  /** Does this class have an arranged number of units?
+   * If true, u1, u2, u3 are set to zero.
+   */
```

Comment:
jsdoc standard is to have a newline after the `/**` for multiline comments

see https://jsdoc.app/

```suggestion
  /**
   * Does this class have an arranged number of units?
   * If true, u1, u2, u3 are set to zero.
   */
```